### DISCUSSION TITLE
Welcome new Maintainer!

### DISCUSSION THREAD

**Discussion by jlooper** [microsoft/Web-Dev-For-Beginners#687] [Announcements]
Dear friends of Web Development!

It's been a real pleasure to have met many of your here online as we learn web development together. I hope you benefited from the repo and took a few more steps down the road toward your goals. I'd like to let you know that I'll be stepping down from the maintainer role of this repo, but you're in great hands as a fellow author, Chris Noring, a true community hero, will be stepping in to manage the repo.

Thank you for helping make this repo the number one most starred non-product repo in Microsoft GitHub!

You are all stars :)

best,
Jen

**Comment by JHBlack** [comment]
Thank you Jen!

**Comment by indcoder** [comment]
Thank you, @jlooper. It has been one of the most useful repository for budding web developers in our community.
Hi @softchris 👋, looking forward to your stewardship.

**Reply by ManuSquall** [reply]
Thanks for the awesome maintainer work you have done here Mrs!

**Comment by Heroo58** [comment]
Can someone tell me how can I start with that and pls don't tell me go to YouTube because I have watched many videos.

**Reply by Suryakanta311** [reply]
Then you buy course or you join a Institute or a industry

**Comment by BigBro17on25tolife** [comment]
Thank You for all your hard work; may you continue to prosper in the
abundance of your skill set, I will keep much and grow from your efforts,
thanks again.

Sincerely
Dana E. Baker; CEO; CSS,CET(a) ETA.
Righteous Neighbor LLC
